easybuild/easyconfigs/p/Perl-bundle-CPAN/Perl-bundle-CPAN-5.36.1-GCCcore-12.3.0.eb

```diff
@@ -2056,22 +2056,6 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/W/WS/WSNYDER'],
         'checksums': ['3973ebdc44682c9c15c776f66e8be242cb4ff1dd52caf43ff446b74d4dccca06'],
     }),
-    ('Sys::Info::Driver::Linux::Device::CPU', '0.7905', {
```

Comment:
Hmm, we should probably retain the top entry here, not the bottom one...

Reply:
Nevermind, same order as in other `Perl-bundle-CPAN` easyconfigs, so should be fine

Reply:
Ah, no, order really does matter, installation of `Sys-Info` is now failing with:
```
Can't locate Sys/Info/Driver/Linux/Device/CPU.pm in @INC
```

I'll fix that in this PR...

Reply:
fixed in 9d8f5e25e1